### DISCUSSION TITLE
Didn't it check if the import path exists

### DISCUSSION THREAD

**Discussion by ahao268** [eslint/eslint#16624] [Ideas] [answered]
when I import path, and this path doesn't exist, but it didn't tell me

**Comment by mdjermanovic** [comment, marked as answer]
> when I import path, and this path doesn't exist, but it didn't tell me

You can use this [`eslint-plugin-import`](https://github.com/import-js/eslint-plugin-import) rule:

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md
